README.md

```diff
@@ -30,7 +30,7 @@ flavor of the STL (native desktop). We need to extend this to build all of the f
 because they need to be updated whenever source files are added/renamed/deleted. We'll delete the legacy machinery as
 soon as possible.)
 
-* Tests: **Coming soon.** We rely on three test suites: devcrt, tr1, and [libcxx][]. We need to replace our current test
+* Tests: **Coming soon.** We rely on three test suites: std, tr1, and [libcxx][]. We need to replace our current test

```

Comment:
Technically we also now rely on vcr, though that won't be published here because that's where we put the stuff with licensing scary. Do we want to mention that?

Reply:
I think this should just be for things that we expose (or potentially intend to expose) externally.

Reply:
I don't think we need to mention internal-only test suites.